Rakefile

```diff
@@ -25,7 +25,8 @@ Rake::Task[:coverage].clear
 
 # Remove puppetlabs_spec_helper's metadata and validate tasks
 Rake::Task[:validate].clear
-Rake::Task[:metadata].clear
+# This was removed in puppetlabs_spec_helper 1.1.0
+Rake::Task[:metadata].clear if Rake::Task.task_defined?(:metadata)
```

Comment:
I feel like [this](https://github.com/praekeltfoundation/puppet-docker_firewall/commit/708e37fa7ff23173775e54814f0dad86adcc9696) approach is slightly more robust?